.github/workflows/create_release.yml

```
@@ -18,7 +18,7 @@ jobs:
       - id: get-release-vars
         name: Configure Release Vars
         run: |
-          echo release_version=v$(grep -E "version=\"[0-9]+\.[0-9]+\.[0-9]+\"" setup.py | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+") >> "$GITHUB_OUTPUT"
+          echo release_version=v$(grep -E "version=\"[0-9]+\.[0-9]+\.[0-9]+\"" pyproject.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+") >> "$GITHUB_OUTPUT"
```

There are whitespaces around the equality sign in `pyproject.toml`. So I think
```suggestion
          echo release_version=v$(grep -E "version = \"[0-9]+\.[0-9]+\.[0-9]+\"" pyproject.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+") >> "$GITHUB_OUTPUT"
```
would work better.

---

Though, personally, I prefer using Python over shell pipes for more complex conversions whenever possible.

Here's my way of doing this: https://github.com/codecov/codecov-cli/pull/604/files#r1938056287. Even though, it's longer, the readability is improved greatly, plus regexes are avoided in favor of native parsing.